bindings/wasm/test/examples/auger.mjs

```diff
@@ -3,7 +3,7 @@ const beadRadius = 2;
 const height = 40;
 const twist = 90;
 
-import {CrossSection, Manifold, setMinCircularEdgeLength} from 'manifold-3d/manifoldCAD';
```

Comment:
Why don't we need to import this anymore? Seems to go against the idea of being explicit.